services/auth.js

```diff
@@ -1,7 +1,7 @@
 const { OAuth2Client } = require('google-auth-library')
 const tokenService = require('~/services/token')
 const emailService = require('~/services/email')
-const { getUserByEmail, createUser, updateUser, getUserById } = require('~/services/user')
+const { getUserByEmail, createUser, _updateUser, getUserById } = require('~/services/user')
```

Comment:
is it necessary to use _ before updateUser?

Reply:
i have marked it as private service function

Reply:
I am not sure we need that.

Reply:
so how do you think it would be good to call a function of service that won't be used for performing after request on endpoint, cause for this there is a function with filtration of incoming fields?